docs/WALKTHROUGH.md

```diff
@@ -0,0 +1,408 @@
+# Service Catalog Demonstration Walkthrough
+
+This document outlines the basic features of the service catalog by walking
+through a basic use case.
+
+## Step 0 - Prerequisites
+
+### Starting Kubernetes with DNS
+
+You *must* have a Kubernetes cluster with cluster DNS enabled.
+
+* If you are using Google Container Engine or minikube, no additional action is
+required.
+* If you are using hack/local-up-cluster.sh, ensure to set
+KUBE_ENABLE_CLUSTER_DNS:
+
+```console
+KUBE_ENABLE_CLUSTER_DNS=true hack/local-up-cluster.sh -O
+```
+
+### Getting Helm and installing Tiller
+
+If you already have Helm v2 or newer, executing `helm init` should be all that's
+required. Detailed helm installation instructions are available
+[here](https://github.com/kubernetes/helm/blob/master/docs/install.md).
```

Comment:
This almost makes it sound like Helm 2+ is recommended for the sake of convenience only. I would state more firmly that Helm 2+ is required. The charts aren't at all compatible with Helm Classic.